components/brave_wallet_ui/page/screens/send/components/select_address_modal/select_address_modal.tsx

```diff
@@ -0,0 +1,795 @@
+                      fevmTranslatedAddresses?.[account.accountId.address]
+                    }
+                  />
+                ))}
+              </>
+            )}
+            {isSearchingForDomain && (
+              <Row margin='26px 0px 0px 0px'>
+                <DomainLoadIcon />
+                <Text
+                  textSize='14px'
+                  isBold={false}
+                  textColor='secondary'
+                >
+                  {getLocale('braveWalletSearchingForDomain')}
+                </Text>
+              </Row>
+            )}
+            {filteredAccounts.length === 0 &&
+              !isSearchingForDomain &&
+              !showEnsOffchainWarning &&
+              addressMessageInformation?.type !== 'error' && (
+                <AddressButton onClick={() => onSelectAddress(searchValue)}>
+                  <WalletIcon />
+                  <Column alignItems='flext-start'>
+                    <AddressButtonText
+                      textSize='14px'
+                      isBold={true}
+                      textColor='primary'
+                      textAlign='left'
+                    >
+                      {searchValue}
+                    </AddressButtonText>
+                    {searchValueHasValidExtension && (
+                      <AddressButtonText
+                        textSize='14px'
+                        isBold={false}
+                        textColor='secondary'
+                        textAlign='left'
+                      >
+                        {resolvedDomainAddress}
+                      </AddressButtonText>
+                    )}
+                  </Column>
+                </AddressButton>
+              )}
+            {showAddressMessage && (
+              <>
+                <VerticalSpace space='8px' />
+                <AddressMessage
+                  addressMessageInfo={addressMessageInformation}
+                  onClickEnableENSOffchain={
+                    addressMessageInformation.id ===
+                    AddressMessageInfoIds.ensOffchainLookupWarning
+                      ? onENSConsent
+                      : undefined
+                  }
+                  onClickHowToSolve={
+                    addressMessageInformation.id ===
+                      AddressMessageInfoIds.invalidChecksumError ||
+                    addressMessageInformation.id ===
+                      AddressMessageInfoIds.missingChecksumWarning
+                      ? () => setShowChecksumInfo(true)
+                      : undefined
+                  }
+                />
+              </>
+            )}
+          </ScrollContainer>
+        </Wrapper>
+      </PopupModal>
+    )
+  }
+)
+
+function getAddressMessageInfo({
+  fevmTranslatedAddresses,
+  showFilecoinFEVMWarning,
+  toAddressOrUrl,
+  messageId,
+  coinType
+}: {
+  showFilecoinFEVMWarning: boolean
+  fevmTranslatedAddresses:
+    | Map<string, { address: string; fvmAddress: string }>
```

Comment:
Should this be a Record?
```suggestion
    | Record<string, { address: string; fvmAddress: string }>
```

Reply:
`GetFVMAddressResult`  returns a type `Map<string, { address: string; fvmAddress: string }>`
https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/common/slices/endpoints/address.endpoints.ts#L26

Reply:
ok, ill follow up to fix this